src/v/model/tests/BUILD

```diff
@@ -74,6 +74,23 @@ redpanda_cc_btest(
     ],
 )
 
+redpanda_cc_btest(
+    name = "timeout_adl",
```

Comment:
I know the cmake test doesn't set cpu=1 but this test only uses a single core - can we set that for Bazel?